taxonomies/labels.txt

```diff
@@ -19396,8 +19396,9 @@ pt:Maçãs da França
 origins:en: en:france
 ingredients:en: en:apple
 
-en:I protect the bees
-fr:Je protège les abeilles
+en:bee friendly, I protect the bees
```

Comment:
They capitalize their logo: BEE FRIENDLY. It's a bit harsh, I think we could use "Bee Friendly" as the main capitalization (which will be displayed in the lists of labels).

Reply:
Fixed, thanks.